# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/nsBaseWidget.cpp
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 widget/nsBaseWidget.cpp
--- a/widget/nsBaseWidget.cpp	Sun May 26 14:29:56 2019 +0000
+++ b/widget/nsBaseWidget.cpp	Sun May 26 14:30:14 2019 +0000
@@ -48,10 +48,10 @@
 #include "mozilla/Attributes.h"
 #include "mozilla/Unused.h"
 #include "nsContentUtils.h"
-#include "gfxPrefs.h"
 #include "mozilla/gfx/2D.h"
 #include "mozilla/MouseEvents.h"
 #include "GLConsts.h"
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/Unused.h"
 #include "mozilla/IMEStateManager.h"
 #include "mozilla/VsyncDispatcher.h"
@@ -857,7 +857,7 @@
       NewRunnableMethod<float>("layers::IAPZCTreeManager::SetDPI", mAPZC,
                                &IAPZCTreeManager::SetDPI, dpi));
 
-  if (gfxPrefs::APZKeyboardEnabled()) {
+  if (StaticPrefs::APZKeyboardEnabled()) {
     KeyboardMap map = nsXBLWindowKeyHandler::CollectKeyboardShortcuts();
     // On Android the main thread is not the controller thread
     APZThreadUtils::RunOnControllerThread(NewRunnableMethod<KeyboardMap>(
